module3/homework/inner_product/arithmeticAverage.cpp

```diff
@@ -0,0 +1,14 @@
+#include "arithmeticAverage.hpp"
+
+#include <cmath>
+#include <functional>
+#include <numeric>
+
+
+float ArithmeticAverage::operator()(std::vector<int>& lhs, std::vector<int>& rhs) const
+{
+    const int initNum = 0;
+    int fullSize = lhs.size() + rhs.size();
+    auto sum = std::fabs(std::inner_product(lhs.begin(), lhs.end(), rhs.begin(), initNum, std::plus<>(), std::plus<>()));
```

Comment:
I suggest you to use const iterators whenever you can

Reply:
+1